docs/source/intro-clx-dga.ipynb

```diff
@@ -127,7 +127,9 @@
      "name": "stderr",
```

Comment:
<p>prediction is different here. is this expected?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Because we're demonstrating model training rather than actual training, this is to be expected.</p>